webpack.config.js

```diff
@@ -63,7 +63,7 @@ switch (nodeEnvironment) {
       bundle: './index.js',
       vendor: ['angular', 'angular-ui-router', 'lodash']
     }
-
+    config.devtool = 'source-map';
```

Comment:
Instead of having devtool defined in two places on the config object in the switch statement. Define it outside of the case above so that it pulls that property no matter what.